pkgs/development/lisp-modules/COPYING

```diff
@@ -0,0 +1,22 @@
+Copyright 2021-2023 Kasper Gałkowski
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
```

Comment:
I am tempted to ask, if there is an upstream repo for this code, and the license itself is different from Nixpkgs, is it a good idea to just _package_ `nix-cl`.

If not, I think it is better to create a subdirectory for the generator and put the files there. Both from the point of view of not mixing the generator codebase with the output, and also to be able to separate the Nix-evaluation-time files from this licensing (presumably we will get to three or five people editing the output side pretty quickly)

Reply:
I can remove this COPYING file, then Nixpkgs license will apply

Reply:
The code for importing from Quicklisp is already in the separate `import` directory

Reply:
Ah right, indeed the builder is another story.

I think as is the situation with the COPYING file is going towards implying false claims, so it needs fixing. I guess either packaging-from-upstream-repo or letting match Nixpkgs are perfect if you are OK with either, otherwise it's fine to isolate into subdirs so that COPYING would cover something clearly separated from things intended to be immediately edited by others.

Reply:
I think I will just remove the COPYING here. 

But I'm not sure from your comment, do you think removing the COPYING file here will solve the problem?

Reply:
I think so. Nixpkgs has various pieces of code contributed for inclusion into the Nixpkgs under the current license, you are the sole author so you can choose to submit for inclusion under MIT license. And Nixpkgs license does refer to contributors as people coauthoring Nixpkgs, instead of listing specific people by name. I guess the end user will have a choice of taking your code from BSD2 upstream or from MIT downstream.

I was just against «put COPYING where it seems to cover a file that will be edited by people not listed in closed-list-of-people COPYING», and also a bit against having BSD2-licensed Nix expressions in MIT-licensed Nixpkgs.